targets/tizenrt-artik05x/configs/jerryscript/Make.defs

```diff
@@ -1,5 +1,37 @@
+###########################################################################
+# Copyright JS Foundation and other contributors, http://js.foundation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+###########################################################################
+#
+# Copyright 2017 Samsung Electronics All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
```

Comment:
@akosthekiss I got it. You don't want to keep NuttX license in jerryscript project since the project only permits Apache2 license. Then, I have another question. What is the difference between this case and https://github.com/jerryscript-project/jerryscript/tree/master/third-party/valgrind/memcheck.h (or valgrind.h)? If these are Apache2 compatible licenses, NuttX seems to be also compatible when I take a close look into each clauses.